docs/webhooks.md

```diff
@@ -0,0 +1,34 @@
+### Webhooks
```

Comment:
I think it's important to mention here the webhooks websocket should not be exposed to the public web for security reasons. We don't have a websocket endpoint with authentication enabled right?

Reply:
good point. I mention that this shouldnt be exposed to the ww at all in the workflows doc but you're right. good to be explicit and mention it here